src/lib/y2storage/storage_manager.rb

```diff
@@ -339,6 +339,29 @@ def mode
       environment.read_only? ? :ro : :rw
     end
 
+    # Whether there is any device in the system that may be used to install a
+    # system.
+    #
+    # This method does not check sizes or any other property of the devices.
+    # It performs a very simple check and returns true if there is any device
+    # of one of the acceptable types (basically disks or DASDs).
+    #
+    # It will never trigger a hardware probing. The method works even if
+    # such probing has not been performed yet.
+    #
+    # @return [Boolean]
+    def devices_for_installation?
+      if probed?
+        !probed.empty?
```

Comment:
probed.empty? is different from Storage.light_probe. E.g. the first is ```true``` if only NFS mount points are found, e.g. of the installation medium, while the second one is ```false```.

Reply:
Well, if we want a closer behavior in both cases we could check for `probed.disk_devices.empty?`

Although we would need to revisit the code after adding support for Xen virtual partitions. https://trello.com/c/EqN6jm0m/370-sles15-p3-1085134-cannot-install-when-using-xen-virtual-partitions-xvda1xvda2